src/main/java/seedu/address/logic/commands/ShowStatisticCommand.java

```diff
@@ -1,29 +1,18 @@
 package seedu.address.logic.commands;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.expense.Category;
 
 /**
  * Show the statistic and brief summary of the user's expenses.
  */
 public class ShowStatisticCommand extends Command {
 
-    public static final String COMMAND_WORD = "showStatistic";
+    public static final String COMMAND_WORD = "showstatistics";
```

Comment:
Can I double confirm this with our team that using upper case alphabet for second word of the command? 🤔 
Someone change my list command to use uppercase alphabet for second word and not using all lower case alphabet.